services/src/main/scala/cromwell/services/cost/GcpCostCatalogService.scala

```diff
@@ -116,6 +136,9 @@ object GcpCostCatalogService {
       s"Expected usage units of RAM to be 'GiBy.h'. Got ${usageUnit}".invalidNel
     }
   }
+
+  // TODO: implement this
+  def calculateGpuPricePerHour(gpuSku: Sku, gpuCount: Long): ErrorOr[BigDecimal] = BigDecimal(1).validNel
```

Comment:
I did not have time to dig into how `calculateCpuPricePerHour` and `calculateRamPricePerHour` work, but I expect this method's implementation should be similar. At a high level, we need to get the pricing info from the SKU, convert it to cost per GPU per hour, and then multiply by the GPU count to get the total GPU price per hour.